core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -105,7 +105,7 @@ protected function doReplace($strBuffer, $blnCache)
 		for ($_rit=0, $_cnt=\count($tags); $_rit<$_cnt; $_rit+=2)
 		{
 			$strBuffer .= $tags[$_rit];
-			$strTag = $tags[$_rit+1];
+			$strTag = $tags[$_rit+1] ?? ''; // todo: is this correct? than we should prob backport it
```

Comment:
Yes, IMO this is correct because `null == ''` and therefore line 111 would always `continue` (or effectively `break` in this case).

Reply:
Should we fix this in 4.4./4.9? I mean, there is lots of 'invalid array access' in the legacy code and it's only showing up because I'm calling it from a test that runs in a stricter mode. :see_no_evil:

Reply:
Only fix it here IMO. As you said, it’s only necessary for the tests...

Reply:
Removed comment in d3e374504dd6ddc99fac6c8a4650b9e0c18d5f09.